src/databases/models/FCMToken.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const {Model} = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class FCMToken extends Model {}
+
+  FCMToken.init(
+    {
+      user_id: {type: DataTypes.UUID, allowNull: false, primaryKey: true},
+      token: {type: DataTypes.STRING, allowNull: true, unique: true},
+      created_at: {type: DataTypes.DATE, allowNull: false},
+      updated_at: {type: DataTypes.DATE, allowNull: false}
+    },
+    {
+      sequelize,
+      modelName: 'FcmToken',
+      tableName: 'user_token',
+      underscored: true
+    }
+  );
+  return FCMToken;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The model definition for `FCMToken` looks good. However, I noticed that the `token` field is set to allow null values (`allowNull: true`). If this token is essential for sending notifications, it should not be nullable. Consider changing `allowNull` to `false` for the `token` field.
```diff
-      token: {type: DataTypes.STRING, allowNull: true, unique: true},
+      token: {type: DataTypes.STRING, allowNull: false, unique: true},
```
Also, Sequelize automatically adds `createdAt` and `updatedAt` fields to your model, so you don't need to define them manually unless you want to customize their names or types. You can remove lines 11 and 12.
```diff
-      created_at: {type: DataTypes.DATE, allowNull: false},
-      updated_at: {type: DataTypes.DATE, allowNull: false}
```
Lastly, ensure that the table name 'user_token' in line 17 matches the actual table name in your database.
```diff
-      tableName: 'user_token',
+      tableName: 'your_actual_table_name',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->